lib/bandit.ex

```diff
@@ -71,6 +71,8 @@ defmodule Bandit do
   * `scheme`: One of `:http` or `:https`. If `:https` is specified, you will need
      to specify `certfile` and `keyfile` in the `transport_options` subsection of `options`.
      Defaults to `:http`
+  * `:startup_log` - The log level at which Bandit should log startup info.
```

Comment:
I've updated naming to `startup_log`. It seemed better than `log_startup` since it's the log level, not boolean.